hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/server/StorageContainerManager.java

```diff
@@ -1010,6 +1011,9 @@ public void stop() {
     } catch (Exception ex) {
       LOG.error("SCM Metadata store stop failed", ex);
     }
+
+    // shutdown RatisPipelineUtils pool.
+    RatisPipelineUtils.POOL.shutdown();
```

Comment:
shutdown will wait for previously submitted tasks to complete. You can probably call shutdownNow since we don't care about the completion on shutdown.

Reply:
Done